glide-core/redis-rs/redis/src/client.rs

```diff
@@ -86,6 +86,8 @@ pub struct GlideConnectionOptions {
     #[cfg(feature = "aio")]
     /// Passive disconnect notifier
     pub disconnect_notifier: Option<Box<dyn DisconnectNotifier>>,
+    /// True if ReadFromReplica strategy is set to AzAffinity
```

Comment:
You should add a comment about the variable, not about when it is set to `true`.

For example:

```rust
/// When set to `true`, the connection will trigger an `INFO` command and search for the node's `availability_zone` property.
/// This value is then kept for trying to find the best replica for routing read-only commands (based on the `ReadFromReplica` strategy)
```